src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-size: center;
```

Comment:
The 'background-size' property should have a value such as 'contain' or a specific size (e.g., '16px 16px') to properly size the background image. The current value 'center' is not a valid value for this property.